airflow/example_dags/example_external_task_marker.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
```

Comment:
Let's change filename to `airflow/example_dags/example_external_task_marker_dag.py`

Reply:
Although it is redundant but that is the convention of other dags too